.vsts-dotnet-ci.yml

```diff
@@ -67,6 +70,7 @@ jobs:
     vsTestConfiguration: "/Framework:.NETCoreApp,Version=v3.0"
     pool:
       name: Hosted VS2017
+    helixQueue: Windows.10.Amd64.Open
```

Comment:
Why not rather than duplicating these values, set the queue on `job-template` based on the pool name or other parameter. The container based runs can get complicated and it might be simpler to have all the queues at one location and have logic to decide when to use which.

See in `dotnet/runtime` is a little bit more complex but is nice to avoid duplicating queues: https://github.com/dotnet/runtime/blob/main/eng/pipelines/libraries/helix-queues-setup.yml

Reply:
Do you think all the extra complexity with the conditionals will be beneficial in our case? We don't have nearly as many queues or situations as the runtime does. We only set one queue per job, does that change anything?

If you think we should still do it lets sync tomorrow on this. I'm not super familiar with the yml format for AzDo and it will be much faster if we do it together.

Reply:
Maybe for your scenario the extra complexity is not worth it. If you start adding more and more queues, maybe consider revisiting.

Reply:
I will be adding a few more. For arm/arm64/m1. For those though we will only have a debug/release build for each. Is that enough you think to make it better to add in this logic?

Reply:
I don't have a strong opinion. If you ever feel like it is getting too much repetition might be worth exploring the other option.